### PR TITLE
A proposal for a vibration/rumble API.

### PR DESCRIPTION
**Pull request by Perksey**
# Summary of the PR
- Adds vibration to gamepads
- Supports multiple vibration motors, e.g for HD Rumble.

# Related issues, Discord discussions, or proposals
#38 

# Further Comments

### CODE REVIEW

File: documentation/proposals/Proposal - Rumble.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+### IRumble
+```cs
+public interface IMotor
+{
+    /// <summary>
+    /// The index of this vibration motor.
+    /// </summary>
+    int Index { get; }
+    /// <summary>
+    /// How fast/intense this motor is moving/vibrating, between 0f and 1f.
```
Comment (frederikja163) [inline documentation/proposals/Proposal - Rumble.md]: I dont like all the slashes, i think we should stick to one word, ie. `/// How intensive this motor is vibrating, between 0f and 1f.
`

File: documentation/proposals/Proposal - Rumble.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Summary
+
+Proposal API for gamepad vibration/rumble control.
```
Comment (frederikja163) [inline documentation/proposals/Proposal - Rumble.md]: I think we should stick to one word, ie. `Proposal API for gamepad vibration control. `

File: documentation/proposals/Proposal - Rumble.md (hunk 1 above)
Comment (Perksey) [reply documentation/proposals/Proposal - Rumble.md]: Fixed.

File: documentation/proposals/Proposal - Rumble.md (hunk 2 above)
Comment (Perksey) [reply documentation/proposals/Proposal - Rumble.md]: Fixed.

**Pull request merged**